packages/design-system/src/components/Pagination/Pagination.test.tsx

```diff
@@ -31,8 +31,7 @@ function queryNextLink() {
 }
 
 function hasEllipsis() {
-  const items = screen.getAllByRole('listitem');
-  return items.some((item) => item.querySelector('.ds-c-pagination__overflow'));
+  return document.querySelector('.ds-c-pagination__overflow');
```

Comment:
I know testing library prefers you use their API to query component structure, but the changes I made removed the list role items in favor of generic role items and there's no easy way to query that part of the markup without adding something like data-testId attributes everywhere.